src/sentry/dynamic_sampling/tasks/helpers/boost_low_volume_projects.py

```diff
@@ -28,6 +28,9 @@ def get_boost_low_volume_projects_sample_rate(
 def get_boost_low_volume_projects_sample_rate(
     org_id: int, project_id: int, *, error_sample_rate_fallback: float | None
 ) -> tuple[float | None, bool]:
+    # Returned tuple contains
+    # - the sample rate as a float
+    # - a boolean specifying whether a sample rate or fallback was used for the sample rate (False -> fallback was used)
```

Comment:
nit: if you make this a python docstring, it will be picked up by IDE when hovering the function name, so it will be easier to access this comment

Reply:
nice, changed it